content/pages/1.install.md

```diff
@@ -39,15 +43,21 @@ The easiest way install ``mycli`` on a Mac is to use [homebrew].
     $ sudo apt-get install mycli
 ```
 
-### RHEL, Centos, Fedora:
+### Fedora
+
+```
+    $ sudo dnf install mycli
+```
+
+### RHEL, Centos:
 
-I haven't built an RPM package for mycli yet. So please use `pip` to install `mycli`. You can install pip on your system using:
+We don't have an RPM package for mycli, yet. So, please use `pip` to install `mycli`. You can install pip on your system using:
```

Comment:
If people are able to install it on Fedora doesn't that mean there is an RPM package for it? 

/cc @meeuw